themes/_default.yaml

```diff
@@ -22,37 +24,38 @@ generic:
       font-family: "Handlee"
       generic-family: "cursive"
   color:
-    accent-darker: &accent-darker   "#0B4848"
-    accent-dark: &accent-dark       "#177D7D"
-    accent: &accent                 "#2A9595"
-    accent-border: &accent-border   "#E6BB00"
-    accent-bottom: &accent-bottom   "#FFD05E"
-    accent-light: &accent-light     "#46BCBC"
-    accent-lighter: &accent-lighter "#6EE1E1"
-    accent-top: &accent-top         "#FFD97C"
+    accent: &accent                             "#56B99D"
+    accent-border: &accent-border               "#3E8A74"
+    accent-border-hover: &accent-border-hover   "#177D7D"
+    accent-bottom: &accent-bottom               "#FFD05E"
+    accent-dark: &accent-dark                   "#4baf93"
+    accent-darker: &accent-darker               "#0B4848"
+    accent-light: &accent-light                 "#46BCBC"
+    accent-lighter: &accent-lighter             "#6EE1E1"
+    accent-top: &accent-top                     "#FFD97C"
 
-    primary-darker: &primary-darker     "#162052"
-    primary-dark: &primary-dark         "#283A8E"
-    primary-dark-rgb: &primary-dark-rgb "40,58,142"
-    primary: &primary                   "#3F52A9"
-    primary-light: &primary-light       "#7F92E7"
-    primary-lighter: &primary-lighter   "#5B6ECA"
+    primary-darker: &primary-darker             "#979ea3"
+    primary-dark: &primary-dark                 "#283A8E"
+    primary-dark-rgb: &primary-dark-rgb         "40,58,142"
+    primary: &primary                           "#3F52A9"
+    primary-light: &primary-light               "#7F92E7"
+    primary-lighter: &primary-lighter           "#5B6ECA"
 
-    secondary-darkest: &secondary-darkest "#57410f"
-    secondary-darker: &secondary-darker   "#785A13"
-    secondary-dark: &secondary-dark       "#CF9E26"
-    secondary: &secondary                 "#F8C346"
-    secondary-rgb: &secondary-rgb         "248,195,70"
-    secondary-light: &secondary-light     "#FFD05E"
-    secondary-lighter: &secondary-lighter "#FFD97C"
+    secondary-darkest: &secondary-darkest       "#A5AEB4"
+    secondary-darker: &secondary-darker         "#BEC3C7"
+    secondary-dark: &secondary-dark             "#dbdee0"
+    secondary: &secondary                       "#F8C346"
+    secondary-rgb: &secondary-rgb               "248,195,70"
+    secondary-light: &secondary-light           "#FFD05E"
+    secondary-lighter: &secondary-lighter       "#FFD97C"
 
     text-dark: &text-dark "#333"
     text: &text "#666"
     text-light: &text-light "#999"
     text-lighter: &text-lighter "#998"
     text-lighter-rgb: &text-lighter-rgb "153,153,136"
 
-    active: &active "#3892e7"
+    active: &active                             "#4baf93"
```

Comment:
I think something went wrong here   =]

Reply:
@froskie depending on what alignment standards you use, some variables align by colors, some variables no) for example: 
![screen shot 2017-10-23 at 15 57 42](https://user-images.githubusercontent.com/16314636/31893168-2259d3fe-b80b-11e7-9b96-00b4c64a86e0.png)
How it should be aligned?